components/camel-casper/src/main/java/org/apache/camel/component/casper/CasperConstants.java

```diff
@@ -0,0 +1,73 @@
+package org.apache.camel.component.casper;
+
+/**
+ * Constants
+ * @author mabahma
+ */
+public interface CasperConstants {
```

Comment:
Please use a class to define the constants instead of using an interface.

Reply:
fixed